userDAO.js

```diff
@@ -0,0 +1,15 @@
+// development dependencies
+var app = require('express')();
+var DA0 = require('../DOAfactory');
+var db = require('../..database');
+
+db.connect();
+
+var UserDAO = DAO("UserDAO", client,
+  (
+    firstName TEXT(25) NOT NULL,
```

Comment:
Expected ')' to match '(' from line 9 and instead saw 'TEXT'.<br>Expected ')' and instead saw 'NOT'.<br>Missing semicolon.<br>'firstName' is not defined.<br>'NULL' is not defined.